examples/consume/app.ts

```diff
@@ -24,7 +24,7 @@ const consume = async (consumer: Consumer) => {
     }
 
     // We commit every message
-    await message.commit()
+    // await message.commit()
```

Comment:
Why are you doing this change in this MR? I cannot see how this is related

Reply:
done